testsuite/unittests/test_rdv_param_setting.sli

```diff
@@ -244,20 +244,39 @@ statusdict /have_gsl :: {
   /MT19937 /uniform_int << /low 11 /high 10 >> run_test
 } fail_or_die
 
+/:range_limit
```

Comment:
@heplesser My problem is the function :range_limit 
Can you add a comment, explaining what parameter it expects and what it is exactly computing in an overflow-safe way?